rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -875,16 +885,6 @@ size_t Pir2Rir::compileCode(Context& ctx, Code* code) {
                 EMPTY(PirCopy);
 #undef EMPTY
 
-            case Tag::IsObject: {
```

Comment:
This was my fault. I left them because @o- did not like the fusing of the is instructions into a typetest and he was planning to rollback